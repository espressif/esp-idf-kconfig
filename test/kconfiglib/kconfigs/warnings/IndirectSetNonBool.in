mainmenu "Indirectly set values in non-bool configs"

    config NONBOOL
        int "Non-bool config"
        default 0
        set A=1
        set default A=42
        select B
        imply C

    config A
        int "A"
        default 0

    config B
        bool "B"
        default n

    config C
        bool "C"
        default n
