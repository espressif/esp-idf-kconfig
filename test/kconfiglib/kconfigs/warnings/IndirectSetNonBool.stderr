warning: NONBOOL of type int
has 'select' option, which is only supported for boolean symbols. Option ignored.
has 'imply' option, which is only supported for boolean symbols. Option ignored.
has 'set' option, which is only supported for boolean symbols. Option ignored.
has 'set default' option, which is only supported for boolean symbols. Option ignored.
